apps/i18n/codebridge/en_us.json

```diff
@@ -8,27 +7,29 @@
   "cannotStop": "We don't know how to stop your code. Please contact support.",
   "cannotTest": "We don't know how to run your tests. Please contact support.",
   "defaultLayout": "Default layout",
+  "defaultNewFileContents": "Add your changes to {fileName}",
   "delete": "Delete",
-  "deleteFile": "Delete file",
+  "deleteFile": "Delete",
   "deleteFileConfirm": "Are you sure you want to delete the file \"{filename}\"?",
-  "deleteFolder": "Delete folder",
+  "deleteFolder": "Delete",
   "deleteFolderConfirm": "Are you sure you want to delete the folder \"{folderName}\"?",
   "deleteFolderConfirmBoth": "This will also delete {fileCount} files and {folderCount} subfolders inside \"{folderName}\".",
   "deleteFolderConfirmFiles": "This will also delete {fileCount} files inside \"{folderName}\".",
   "deleteFolderConfirmSubfolders": "This will also delete {folderCount} subfolders inside \"{folderName}\".",
-  "downloadFile": "Download file",
+  "downloadFile": "Download",
   "downloadFileError": "Cannot download files of type {extension}",
   "duplicateFileError": "Filename {fileName} is already in use in this folder. Please choose a different name.",
   "duplicateFolderError": "Foldername {folderName} is already in use in this folder. Please choose a different name.",
   "duplicateSupportFileError": "Filename {fileName} is already in use in this folder in the level's support code. Please choose a different name.",
   "folderExistsError": "Folder already exists with that name",
-  "invalidNameError": "Names must be only letters, numbers, or dots",
+  "invalidFileUpload": "Cannot upload files of type {fileType}",
+  "invalidNameError": "Names must be only letters, numbers, or underscores",
   "loadingEnvironmentTooltip": "Your environment is loading. Please wait.",
   "makeStarter": "Make starter file",
   "makeSupport": "Make support file",
   "makeValidation": "Make validation file",
-  "moveFile": "Move file",
-  "moveFolder": "Move folder",
+  "moveFile": "Move",
```

Comment:
should we consolidate these into one string? It will be consistently translated if we do that (same for delete and rename)

Reply:
I think we should keep them separate for now at least, but it'd probably be worth following up with product/design to confirm.

I like having them separate so it's trivial to change the display text for each individually, but if we're completely guaranteed not to do that, we should consolidate. I think it's a nice-to-have, after confirming.